app/jobs/noticed/event_job.rb

```diff
@@ -1,7 +1,5 @@
 module Noticed
-  class EventJob < ApplicationJob
-    queue_as :default
```

Comment:
This is [already](https://github.com/rails/rails/blob/a11f0a63673d274c59c69c2688c63ba303b86193/activejob/lib/active_job/queue_name.rb#L9) the default queue name, but by specifying it here, it means users can't change it at the Rails config level.